docs/deployment.uml

```diff
@@ -0,0 +1,201 @@
+Some notes
+==========
+- The intent is for this installer to deploy a system which looks exactly like
+one deployed using `kubeadm`, i.e. using the same (or at least highly similar)
+static manifests, cluster `ConfigMap`s, RBAC roles and bindings,...
```

Comment:
OK about the space. For `...`, I prefer ASCII over anything else if possible. If we ever use this document in e.g. a Sphinx setup, then it will render ellipsis where applicable.

Reply:
Ok, fair enough